### DISCUSSION TITLE
Disable mDNSResponder on macOS Sonoma

### DISCUSSION THREAD

**Discussion by Gijeeg86** [DNSCrypt/dnscrypt-proxy#2604] [Q&A] [answered]
I was reading the wiki and under the MacOS section it is suggested to disable the mDNSResponder. From what I know mDNsResponder in not only the unicast system resolver for Mac but it is also vital for Bonjour framework and all the MDNS queries…so my noob question is: how the mdns queries will be solved if I run DNSCrypt service and disable mDNSResponder?
Thanks

**Comment by jedisct1** [comment]
I've no idea. I've never disabled mDNSResponder and never had issues. Maybe this recommendation is outdated.

**Comment by jedisct1** [comment, marked as answer]
It seems to be only relevant if the "Error 127.0.0.1:53: bind: address already in use" message is printed.

**Comment by Gijeeg86** [comment]
Ok…so far so good. Still testing it.
